04_setup_ironic.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -ex
+
+source common.sh
+
+# Get the various images
+source get_images.sh
+
+# ironic dnsmasq and ipxe config
+cp ironic/dnsmasq.conf $IRONIC_DATA_DIR
+cp ironic/dualboot.ipxe ironic/inspector.ipxe $IRONIC_DATA_DIR/html
```

Comment:
Is it possible that `$IRONIC_DATA_DIR/html` does not exist? May be add a trailing `/`?

Reply:
Will do